tools.js

```diff
@@ -535,11 +535,9 @@ module.exports = (function () {
 					banlistTable[subformat.banlist[i]] = subformat.name || true;
 					banlistTable[toId(subformat.banlist[i])] = subformat.name || true;
 
-					var plusPos = subformat.banlist[i].indexOf('+');
 					var complexList;
-					if (plusPos && plusPos > 0) {
```

Comment:
Alright, I didn't want to make the assumption that `> 0` (as opposed to `>= 0`) wasn't important without knowing what the banlists look like.

Updated to use `includes()` instead.